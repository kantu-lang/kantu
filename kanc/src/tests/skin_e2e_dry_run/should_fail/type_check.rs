use super::*;

#[test]
fn expected_term_of_type_type0_or_type1_2000() {
    let path = concat_paths(
        file!(),
        checked_path!(
            "../../sample_code/should_fail/single_file/type_check/illegal_type/forall_output.k"
        ),
    );
    let output =
        get_manifest_path_and_backslash_normalized_output(vec![DUMMY_EXEC_PATH, "--file", &path]);
    insta::assert_debug_snapshot!(output);
}

#[test]
fn illegal_callee_2001() {
    let path = concat_paths(
        file!(),
        checked_path!(
            "../../sample_code/should_fail/single_file/type_check/illegal_callee/forall.k"
        ),
    );
    let output =
        get_manifest_path_and_backslash_normalized_output(vec![DUMMY_EXEC_PATH, "--file", &path]);
    insta::assert_debug_snapshot!(output);
}
