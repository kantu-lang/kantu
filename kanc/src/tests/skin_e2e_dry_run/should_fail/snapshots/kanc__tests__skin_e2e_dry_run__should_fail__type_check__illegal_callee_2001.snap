---
source: src/tests/skin_e2e_dry_run/should_fail/type_check.rs
expression: output
---
"Error: [E2001] A call expression's callee must have a `forall` type, but the callee at <CARGO_MANIFEST_DIR>/src/tests/sample_code/should_fail/single_file/type_check/illegal_callee/forall.k:3:10 has type     Type<1>."
